mappings/net/minecraft/client/world/ClientWorld.mapping

```diff
@@ -14,7 +14,7 @@ CLASS net/minecraft/class_638 net/minecraft/client/world/ClientWorld
 	FIELD field_34804 chunkUpdaters Ljava/util/Deque;
 	FIELD field_35163 simulationDistance I
 	FIELD field_35432 BLOCK_MARKER_ITEMS Ljava/util/Set;
-	FIELD field_3727 netHandler Lnet/minecraft/class_634;
+	FIELD field_3727 networkHandler Lnet/minecraft/class_634;
```

Comment:
Can you fix the constructor parameter as well?

https://github.com/FabricMC/yarn/blob/29f3c7a07edca6483012bdc10c1a638519db7b57/mappings/net/minecraft/client/world/ClientWorld.mapping#L20-L21

Reply:
This line is right beyond the range for patch, so needs a quick edit (can be done on github ui)

Reply:
@haykam821 are you able to quickly fix the arg^ Thanks 👍